Client/Frontend/Home/FxHomeHorizontalCell.swift

```diff
@@ -142,6 +143,12 @@ class FxHomeHorizontalCell: UICollectionViewCell, ReusableCell {
             descriptionContainer.removeArrangedSubview(faviconImage)
             faviconImage.image = nil
         }
+
+        if viewModel.hasTimeToRead {
+            descriptionLabel.text = "\(viewModel.descriptionText) • \(String.localizedStringWithFormat(String.FirefoxHomepage.Pocket.NumberOfMinutes, viewModel.timeToRead)) "
+        } else {
+            descriptionLabel.text = viewModel.descriptionText
+        }
```

Comment:
I think we could pass in the `viewModel.descriptionText` to have the minutes already in it. i.e. `FxHomePocketViewModel` would have the logic to build that string with minutes. So in the end `FxHomeHorizontalCellViewModel` would be kept a little more agnostic of logic and more reusable for upcoming future use cases.

Reply:
Good idea! It'll keep things cleaner. Let me do that now.